src/tsd/GraphHandler.java

```diff
@@ -112,7 +112,7 @@ public void execute(final TSDB tsdb, final HttpQuery query) {
       if (uri.length() < 4) {  // Shouldn't happen...
         uri = "/";             // But just in case, redirect.
       } else {
-        uri = "/#" + uri.substring(3);  // Remove "/q?"
+        uri = "/#" + uri.substring(2);  // Remove "q?"
```

Comment:
This doesn't work, instead of removing `/q?` it removes `/q` and leaves the `?` behind.  I guess we need something along the lines of `uri = uri.replaceFirst("q\?", "#")` or something of that effect.